go.mod

```diff
@@ -10,7 +10,8 @@ require (
 	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/ghodss/yaml v1.0.0
 	github.com/giantswarm/apiextensions/v2 v2.6.2
-	github.com/giantswarm/apiextensions/v3 v3.4.1
+	github.com/giantswarm/apiextensions/v3 v3.6.0
+	github.com/giantswarm/apptest v0.4.2-0.20201103172001-0ef9f68e6299
```

Comment:
I will change this to use a proper release when we release next `apptest` version.